examples/dwarfdump.rs

```diff
@@ -315,22 +373,23 @@ fn dump_entries<R: Reader>(offset: R::Offset,
         }
 
         let mut attrs = entry.attrs();
-        while let Some(attr) = attrs.next().expect("Should parse attribute OK") {
+        while let Some(attr) = attrs.next()? {
             print!("{:indent$}{:27} ", "", attr.name(), indent = indent + 18);
             if flags.raw {
                 println!("{:?}", attr.raw_value());
             } else {
-                dump_attr_value(&attr, &unit, debug_loc, debug_ranges, debug_str);
+                dump_attr_value(&attr, &unit, debug_loc, debug_ranges, debug_str)?;
```

Comment:
It would be nice if we printed the error and continued dumping in cases where the error doesn't prevent it, such as here.